### PR TITLE
Release task

### PR DESCRIPTION
**Pull request by hannesa2**

### CODE REVIEW

**Comment by hannesa2** [comment]
Now it works....

File: .github/workflows/Android-CI-release.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          commit-ish: HEAD
+      - name: Install Android SDK
+        uses: malinskiy/action-android/install-sdk@release/0.1.1
+      - name: Build project
+        run: ./gradlew clean build
+        env:
+          VERSION: ${{ github.ref }}
+      - run: |
+          assetsAAR=$(find . -name *release.aar | while read -r asset ; do echo "-a $asset" ; done)
+          hub release create ${assetsAAR} -m ${{steps.tagger.outputs.tag}} ${{steps.tagger.outputs.tag}}
```
Comment (niqdev) [inline .github/workflows/Android-CI-release.yml:36]: this will attach the artifact to the release?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # ipcam-view ![ipcam-view](images/logo.png)
 
 [![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-ipcam--view-brightgreen.svg?style=flat)](http://android-arsenal.com/details/1/3358)
-[![Download](https://api.bintray.com/packages/niqdev/maven/mjpeg-view/images/download.svg)](https://bintray.com/niqdev/maven/mjpeg-view/_latestVersion)
```
Comment (niqdev) [inline README.md:4]: Are you going to add JitPack in a different PR?

Comment (hannesa2) [reply README.md:4]: It's less work, when you do it. You have the power to edit `README.md` directly

Please add this info https://jitpack.io/#niqdev/ipcam-view/v1.6.0

image

This is the batch
`
[](https://jitpack.io/#niqdev/ipcam-view)`

File: .github/workflows/Android-CI-release.yml (hunk 1 above)
Comment (hannesa2) [reply .github/workflows/Android-CI-release.yml:36]: yes

File: README.md (hunk 2 above)
Comment (niqdev) [reply README.md:4]: yup, I'll take care of that

**Comment by niqdev** [comment]
Adding just a note in case I'll come back to this. It works only by pushing tags, if you create a release from the ui it will fail with

```
Error creating release: Unprocessable Entity (HTTP 422)
Duplicate value for "tag_name"
Error: Process completed with exit code 1.
```

See https://github.com/niqdev/ipcam-view/actions/runs/697292886

**Comment by hannesa2** [comment]
> Adding just a note in case I'll come back to this. It works only by pushing tags, if you create a release from the ui it will 

Of course it wlll fail !
A new tag will create a release, but when you do it vice versa (first a release, then a (automatic) tag) it's already there !

**Comment by hannesa2** [comment]
I see you did it (finally ?) the right way 👍 Triggered by github action

image

**Pull request merged**